### DISCUSSION TITLE
Backup of RustDesk Server OSS (Docker)

### DISCUSSION THREAD

**Discussion by dernippel** [rustdesk/rustdesk#6696] [Q&A] [answered]
I have set up the RustDesk Server OSS Version using Docker.

What do I need to Backup and possibly restore?

I've already managed to backup/insert my private-public-keypair into the container.

Do I need to backup and restore the sqlite3 files, too?

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk-server-pro/discussions/184
